monkey/infection_monkey/monkey.py

```diff
@@ -39,6 +40,7 @@ def __init__(self, args):
         self._exploiters = None
         self._fingerprint = None
         self._default_server = None
+        self._default_server_port = None
```

Comment:
Why is this useful? the only use for this variable seems to be to receive a variable and output to the exact same place

Reply:
I separated port, because port is constant meanwhile IP isn't. Now we form `-s` by getting the ip that monkey will reach + adding island port.